plugins/MantisGraph/core/graph_api.php

```diff
@@ -131,20 +131,32 @@ function graph_status_colors_to_colors() {
  * @param string  $p_title        Title.
  * @param string  $p_series_name  The name of the data series.
  * @param string  $p_color        The bar color.
+ * @param integer $p_wfactor      Width factor for graph chart. Eg: 2 to make it double wide
  * @return void
  */
-function graph_bar( array $p_metrics, $p_title = '', $p_series_name, $p_color = '#fcbdbd' ) {
+function graph_bar( array $p_metrics, $p_title = '', $p_series_name, $p_color = null, $p_wfactor = null ) {
```

Comment:
I don't see that $p_color is used, so why not
```
function graph_bar( array $p_metrics, $p_title = '', $p_series_name, $p_wfactor = 1)
```
and remove 6 lines of the # set defaults code?

Reply:
The thing is: i added that parameter, so i wanted to use:
`graph_bar( metric, title, series, null, wfactor)`
Because, if i don't care about the color parameter, why should i know which color to use?

So i thought that allowing to pass null, meaning default/i_dont_care, seems like a good compromise. 

But since color is no explicitly used by any caller, we could remove it. The color for graphs should be revisited at some point, so that would probably need a different approach anyway